addons/hr_recruitment_integration_base/models/recruitment_platform.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class RecruitmentPlatform(models.Model):
+    _name = 'recruitment.platform'
+    _description = 'Recruitment Platform'
+
+    name = fields.Char(string="Name", required=True)
+    avatar_128 = fields.Binary(string="Logo")
+    website = fields.Char(string="Website")
+
+    # crud pricing
+    currency_name = fields.Char(string="Currency Name", default="Credit(s)")
+    price_to_publish = fields.Float(string="Price to Publish")
+    price_to_get = fields.Float(string="Price to Get")
+    price_to_update = fields.Float(string="Price to Update")
+    price_to_delete = fields.Float(string="Price to Delete")
```

Comment:
What about currencies here ? Those fields should be monetary no ?

Reply:
Yup I'm gonna make a "credit" currency